### PR TITLE
fix(core): fix type error in setElementProperty

### PR DESCRIPTION
**Pull request by DrMarcII**
Convert propertyValue to string when calling setElementAttribute.

### CODE REVIEW

**Comment by TedSander** [comment]
@tbosch Can you take a look? or assign someone better.

File: modules/angular2/src/render/dom/dom_renderer.ts [hunk 1]
```diff
@@ -147,7 +147,7 @@ export class DomRenderer extends Renderer {
     // Reflect the property value as an attribute value with ng-reflect- prefix.
     if (this._reflectPropertiesAsAttributes) {
       this.setElementAttribute(location, `${REFLECT_PREFIX}${camelCaseToDashCase(propertyName)}`,
-                               propertyValue);
+                               `${propertyValue}`);
```
Comment (tbosch) [inline modules/angular2/src/render/dom/dom_renderer.ts]: Could you please add a test for this?

**Comment by DrMarcII** [comment]
@tbosch I added a test that I *think* will detect the problem and verify my solution. PTAL.

**Comment by TedSander** [comment]
Looks like the travis error is valid. The clang formatter should be run?

**Comment by tbosch** [comment]
@DrMarcII Why do you only `think` this? Couldn't you just revert your change and run the test again to see for which error it fails?

**Comment by DrMarcII** [comment]
@tbosch I actually haven't and don't actually know how to run these tests locally. I do know that this fix applied to the generated Dart code does fix the problem.

**Pull request closed without merging**